app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -489,6 +536,59 @@ class StoryFragmentTest {
     }
   }
 
+  /**
+   * Reference:
+   * https://stackoverflow.com/questions/38314077/how-to-click-a-clickablespan-using-espresso
+   */
+  private fun clickClickableSpan(textToClick: CharSequence): ViewAction {
```

Comment:
I found one function, pleae take a look if we can use it.
`openClickableSpan` inside `QuestionPlayerActivityTest`

Reply:
@anandwana001 I used this function in `StoryFragmentTest` and it ran successfully. Should we use this function instead?

Reply:
Yes, we can resume the function what we have already

Reply:
Should I copy the function inside `StoryFragmentTest` or do we need to move it to utility? If we go with the second option then please tell me where should I move this function to.

Thanks.

Reply:
for now, let's copy the function in `StoryFragmentTest` and use it.

Reply:
okay.